ballerina/Module.md

```diff
@@ -1,181 +1,214 @@
 ## Overview
 
-The [Ballerina](https://ballerina.io/) connector for Azure Service Bus allows you to connect to
-an [Azure Service Bus](https://docs.microsoft.com/en-us/azure/service-bus-messaging/) via the Ballerina language.
-
-The Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe topics.It
+The [Azure Service Bus](https://docs.microsoft.com/en-us/azure/service-bus-messaging/) is a fully managed enterprise message broker with message queues and publish-subscribe topics. It
 provides the capability to send and receive messages from Service Bus queues, topics, and subscriptions. The Azure
 Service Bus handles messages that include data representing any kind of information, including structured data encoded
-with the common formats such as the following ones: JSON, XML, Plain Text.
+with common formats such as the following ones: JSON, XML, and Plain Text.
```

Comment:
we can remove the below sentence, which is not that informative IMO
`The Azure
Service Bus handles messages that include data representing any kind of information, including structured data encoded with common formats such as the following ones: JSON, XML, and Plain Text.`

Reply:
Changed with [1660f66](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/206/commits/1660f66747a09bcce86379d2aaff2e35136e6719).